docker/test/performance-comparison/perf.py

```diff
@@ -295,6 +295,9 @@ def do_create(connection, index, queries):
 
     reportStageEnd("create")
 
+# Let's sync the data to avoid writeback affects performance
+os.system('sync')
```

Comment:
perf tests run for an hour at least, so it is questionable if a single sync in the beginning could help

Reply:
It runs each test individually. So sync will be done after each group of CREATE queries

Reply:
got it, then looks good. interesting to see how big will be the overhead